plugins/webp-uploads/settings.php

```diff
@@ -12,11 +12,26 @@
 }
 
 /**
- * Registers setting for generating both JPEG and WebP versions for image uploads.
+ * Registers setting for generating JPEG in addition to the selected modern format for image uploads.
  *
  * @since 1.0.0
+ * @since n.e.x.t The setting was made more general to coveroutputting JPEG as a secondary type. The "webp" option naming
+ *        was left unchanged for backward compatibility.
```

Comment:
Should this indent to match "The" on the line above?

Reply:
right, fixed.